accounts-db/src/accounts_db/geyser_plugin_utils.rs

```diff
@@ -116,25 +113,14 @@ impl AccountsDb {
         &self,
         slot: Slot,
         notified_accounts: &mut HashSet<Pubkey>,
-        mut accounts_to_stream: HashMap<Pubkey, StoredAccountMeta>,
+        accounts_to_stream: HashMap<Pubkey, StoredAccountMeta>,
         notify_stats: &mut GeyserPluginNotifyAtSnapshotRestoreStats,
     ) {
         let notifier = self.accounts_update_notifier.as_ref().unwrap();
         let mut measure_notify = Measure::start("accountsdb-plugin-notifying-accounts");
-        let local_write_version = 0;
-        for (_, mut account) in accounts_to_stream.drain() {
-            // We do not need to rely on the specific write_version read from the append vec.
-            // So, overwrite the write_version with something that works.
-            // 'accounts_to_stream' is already a hashmap, so there is already only entry per pubkey.
-            // write_version is only used to order multiple entries with the same pubkey, so it doesn't matter what value it gets here.
-            // Passing 0 for everyone's write_version is sufficiently correct.
-            let meta = StoredMeta {
-                write_version_obsolete: local_write_version,
-                ..*account.meta()
-            };
-            account.set_meta(&meta);
```

Comment:
i guess this is only updating the local `StoredAccountMeta<'_>`.  It has been 15 months since I last visited this code. I'm happy 0 worked. I don't recall what will happen if we pass random data, which is what is going to happen now.

Reply:
@lijunwangs I'm sorry to have to keep asking you about this.

Reply:
my recollection is that I passed a constant 0 here to completely decouple what was written in the append vecs from what geyser got. Ugh. Are we always zeroing write verison now in master. I think so. But, we aren't zeroing it on 1.17 or 1.18. That means when master loads a snapshot built by 1.17, we'll be passing incrementing write versions here. Maybe that is ok. Maybe that is what we used to do? We may have to wait to remove this until we are always writing 0?
Or, maybe the fn that creates `StoredAccountMeta` is already setting this to 0 everytime? I think I just reviewed that?